front/src/components/Chat/AdminCommands.tsx

```diff
@@ -113,7 +113,7 @@ export function ConfigureChannelPrivate(props: { trigger: boolean, setTrigger: (
 
 	const AddPeoplePrivate = () => {
 		if (allfriend.length > 0)
-			socket.emit('AddPeoplePrivate', { chanid: props.channel.id, users: allfriend });
+			socket.emit('AddPeoplePrivate', { chanId: props.channel.id, users: allfriend.map(user => user.id) });
```

Comment:
'chanId' needs to be 'chanid' to match invite-user-dto
(we can fix it later with the chat-back PR )